### ISSUE TITLE
Feature Request: Allow script to continue when merge conflict (or other error) discovered

### ISSUE THREAD

**Issue by SteveRyan-ASU** [pantheon-systems/terminus-mass-update#4]
When running the following command, the mass-update plugin executes as intended, and works well. 
`terminus site:list --format=list | terminus site:mass-update:apply`

But upon attempting to apply an update where the site has a merge conflict present, the script terminates upon receiving that error.

![2017-02-03_09-41-15](https://cloud.githubusercontent.com/assets/2085753/22599674/ed4ec5fa-e9f4-11e6-8a0c-50615543c8ad.png)

**Potential solutions**
- Allow script to continue executing the remaining items in the queue. Either by default or via another option within the plugin.
- Allow for the `--dry-run` option to test for merge conflicts prior to actually executing the script ahead of time.

Thanks for all that you do! :-)

**Comment by nstielau** [comment]
Good ideas, @SteveRyan-ASU.  Testing for merge conflicts conflicts doesn't seem as useful to me as something like a `--continue-on-failure` flag, and possible spitting out which sites errored at the end.

**Comment by SteveRyan-ASU** [comment]
Agreed about the approach as well. That would also allow the script to continue if it encountered other errors along the way as well. Example: trying to mass-update update a site that was still in SFTP mode would probably also result in the script stopping execution. (Untested.)
